src/analyze/State.re

```diff
@@ -169,7 +169,7 @@ let newJbuilderPackage = (rootPath) => {
   let (otherDirectories, otherFiles) = source |> List.filter(s => s != "." && s != "" && s.[0] == '.') |> optMap(name => {
     let otherPath = rootPath /+ name;
     let res = {
-      let%try jbuildRaw = Files.readFileResult(otherPath /+ "jbuild");
+      let%try jbuildRaw = JbuildFile.readFromDir(otherPath /+ "jbuild");
```

Comment:
`otherPath /+ "jbuild"` -> `otherPath/`

Reply:
Good catch, copy-paste error. Fixed and squashed.